javaee/impl/src/main/java/org/jboss/forge/addon/javaee/cdi/ui/CDINewBeanCommand.java

```diff
@@ -100,6 +101,9 @@ public JavaClassSource decorateSource(UIExecutionContext context, Project projec
       else if (BeanScope.DEPENDENT != scopedValue)
       {
          source.addAnnotation(scopedValue.getAnnotation());
+         if (scopedValue.isSerializable()) {
+            source.addInterface(Serializable.class);
```

Comment:
I think it´s fine to add a serialVersionUID = 1 field to this class. The IDE warnings are a bit annoying to have.
We could introduce a method to calculate the serialVersion based on http://docs.oracle.com/javase/8/docs/platform/serialization/spec/class.html#a4100 in Roaster later